llvm/test/Transforms/MemProfContextDisambiguation/fix_clone_checking.ll

```diff
@@ -0,0 +1,99 @@
+;; Test to make sure we don't fail when cloning in a case where we end up with
+;; a clone that has fewer edges than the node it was initially cloned from.
+;; This test was reduced and simplified from xalancbmk with some random hotness
+;; applied to the profile that reproduced the issue.
+
+; RUN: opt -passes=memprof-context-disambiguation -supports-hot-cold-new \
+; RUNL		-memprof-verify-ccg -memprof-verify-nodes \
```

Comment:
Looks like this line might not have been run in the test? Should be "RUN:" instead of "RUNL".
Since this is the line which enables the verification does it mean the test case reduction was overly aggressive?

Reply:
good catch! I actually had added in the verify options later, because might as well enable that for tests. The failure that was used for test case reduction was elsewhere, because unequal vector lengths passed to allocTypesMatch. Fixing this typo actually caused the test to start failing, because I forgot to also pipe stderr through FileCheck, which I also fixed.